rust/xaynet-core/src/lib.rs

```diff
@@ -1,7 +1,8 @@
 #![cfg_attr(docsrs, feature(doc_cfg))]
 #![doc(
```

Comment:
Do we want to delete the documentation in xaynet-core? 
because xaynet and xaynet-core have now the same documentation
Could be done later (after the release) in separate PR

Reply:
`xaynet-server` has this duplicated docs in its `lib.rs` as well, which could also be removed or replaced by a short wrap-up like for `xaynet-client`.